code/_compile_options.dm

```diff
@@ -77,3 +77,8 @@
 // A reasonable number of maximum overlays an object needs
 // If you think you need more, rethink it
 #define MAX_ATOM_OVERLAYS 100
+
+//Disable unimplemented proc/var warnings
+#ifdef OPENDREAM
+#pragma UnimplementedAccess disable
+#endif
```

Comment:
There are around ~200 of these, very annoying- these just signify that opendream does not implement a given byond builtin